hw/ip/pattgen/data/pattgen.hjson

```diff
@@ -42,12 +42,18 @@
         { bits: "0",
           resval: "0",
           name: "ENABLE_CH0",
-          desc: "Enable pattern generator functionality for Channel 0"
+          desc: "Enable pattern generator functionality for Channel 0",
+          tags: [// don't write (e.g. 1'b1) to enable bit
+                 // because DUT can operate with zero values programmed to registers - it affects several other csrs
+                 "excl:CsrAllTests:CsrExclWrite"]
```

Comment:
Do you need to exclude the `csr_hw_reset` test as well?

Reply:
thanks, I fixed this issue